docs/en/operation/Server-Logging.md

```diff
@@ -136,40 +145,17 @@ have network access to it. In our example, we run the log server on the same mac
 The two environment variables `ALLUXIO_LOGSERVER_HOSTNAME` and `ALLUXIO_LOGSERVER_PORT` control
 the logging behavior of masters and workers in an Alluxio cluster.
 
-Suppose the hostname of the log server is `AlluxioLogServer`, and the port is `45010`.
+Suppose the hostname of the log server is `AlluxioLogServer`, and the port is `45600`.
 The following lines would need to be added to `conf/alluxio-env.sh` to enable remote logging :
 
 ```bash
 ALLUXIO_LOGSERVER_HOSTNAME=AlluxioLogServer
-ALLUXIO_LOGSERVER_PORT=45010
+ALLUXIO_LOGSERVER_PORT=45600
```

Comment:
Changed the example to use default logserver port. I think the previous example makes people think 45010 is the default port.